src/renderer/redux/selectors/discover.js

```diff
@@ -0,0 +1,35 @@
+import { createSelector } from 'reselect';
+import { selectFeaturedUris } from './content';
+import { selectClaimsById, selectAllClaimsByChannel } from './claims';
+
+export const selectDiscover = createSelector(
+  [selectFeaturedUris, selectClaimsById, selectAllClaimsByChannel],
+  (featuredUris, claimsById, claimsByChannel) => {
+    let categories = featuredUris;
+    if (!!categories && !!claimsByChannel) {
+      let channels = [];
+      Object.keys(categories).forEach(key => {
+        if (key.indexOf('@') === 0) {
+          channels.push(key);
+        }
+      });
+      Object.keys(claimsByChannel).forEach(key => {
+        if (channels.includes(key)) {
```

Comment:
you could take out the other forEach loop if you change this line to
`if(categories[key].indexOf('@') === 0)`